### ISSUE TITLE
move to latest ncIdv 4.6.3

### ISSUE THREAD

**Issue by lesserwhirls** [Unidata/IDV#98]
ncIdv-4.6.3 will be stable soon (this week). However, one thing we will run into is that in `ucar.unidata.data.DataManager.initResources()`, the following method from httpclient has changed:

```java
HTTPSession.setGlobalCredentialsProvider(CredentialsProvider provider)
```

no longer works. The IDV will need to use one of the following:

```java
setGlobalCredentialsProvider(AuthScope scope, CredentialsProvider provider)
setGlobalCredentialsProvider(CredentialsProvider provider, String scheme)
setGlobalCredentialsProvider(String url, CredentialsProvider provider)
```
@DennisHeimbigner - any suggestions on which signature should be used here?

**Comment by DennisHeimbigner** [comment]
The second one is the simplest. where
scheme is more or less HTTPAuthSchemes.BASIC

Sean Arms wrote:

**Comment by lesserwhirls** [comment]
fixed in Unidata/IDV#97 - thanks @DennisHeimbigner!
